server/routerlicious/packages/services-core/src/clusterDraining.ts

```diff
@@ -14,3 +14,9 @@ export interface IClusterDrainingChecker {
 	 */
 	isClusterDraining(cluster?: string): Promise<boolean>;
 }
+
+/**
+ * Retry after time when cluster is under draining
+ * @internal
+ */
+export const clusterDrainingRetryTimeInMs = (10 + 2) * 60 * 1000; // 12 minutes
```

Comment:
nit: maybe we can simplify it to just 12 instead of 10 + 2? :)

Reply:
Just want to show that 10 minutes is the default session grace period and 2 minutes is the time buffer.